XamlControlsGallery/ControlPages/NavigationViewPage.xaml.cs

```diff
@@ -237,10 +237,13 @@ private void autoSuggestCheck_Click(object sender, RoutedEventArgs e)
                 AutoSuggestBox asb = new AutoSuggestBox() { QueryIcon = new SymbolIcon(Symbol.Find) };
                 asb.SetValue(AutomationProperties.NameProperty, "search");
                 nvSample.AutoSuggestBox = asb;
+
+                navViewASB.Value = "\r\n    <muxc:NavigationView.AutoSuggestBox> \r\n        <AutoSuggestBox QueryIcon=\"Find\" AutomationProperties.Name=\"Search\" /> \r\n    <" + "/" + "muxc:NavigationView.AutoSuggestBox> \r\n";
```

Comment:
This is set in both here and in the markup above. Are they the same? If so, can they be condensed?

Reply:
No, they can't be condensed. This code is inside the CheckBox's click event. When the page first loads, this event doesn't fire. So I need to initialize the ControlExampleSubstitution in markup, otherwise the appropriate ASB markup won't appear on page load. From code behind, I'm toggling the ControlExampleSubstitution's value between null and the full markup string. The value needs to change depending on the state of the CheckBox and toggle appropriately on click.

Reply:
Gotcha, thanks. I'm mostly concerned about the maintenance of the duplicated markup. Would it be possible to encapsulate this code into a method that is called both in the CheckBox Loaded event and also in the Checkbox handler?

Reply:
Fixed.